libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -111,6 +129,26 @@ class RNBreezSDK: RCTEventEmitter {
             reject(RNBreezSDK.TAG, "Invalid config", nil)
         }
     }
+
+    @objc(signMessage:resolver:rejecter:)
+    func signMessage(_ request: SignMessageRequest, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> SignMessageResponse {
```

Comment:
This reqData type should be `reqData:[String: Any]` as it otherwise causes issues being transferred over the RN bridge